part_b/part_b.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/StephenDsouza90/CollatzConjecture/utils"
+)
+
+// Part B counts the number of steps to reach 1 for each number within a range of numbers.
+func main() {
+	var startValue int
+	var endValue int
+
+	fmt.Println("Enter start value: ")
+	fmt.Scan(&startValue)
+
+	fmt.Println("Enter end value: ")
+	fmt.Scan(&endValue)
+
+	start := time.Now()
+	allSteps := getNumberOfStepsForEachM(startValue, endValue)
+	duration := time.Since(start)
+
+	fmt.Printf("\nAll steps for each number within a range of numbers from %v to %v \n\n", startValue, endValue)
+	fmt.Println(allSteps)
+	fmt.Printf("\n--- Took %v seconds ---\n\n", duration.Seconds())
+}
+
+// getNumberOfStepsForEachM : For each number num within a range of numbers between startValue and endValue,
+// this function returns the number of steps.
+func getNumberOfStepsForEachM(startValue, endValue int) map[int]int {
+	// startValue : starting number of the range
+	// endValue : ending number of the range
+
```

Comment:
This comment is a duplication. It's already explained in the function definition.

Reply:
Removed comment as it is not necessary and easily understood.